OracleLinux/9/scripts/install.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2018 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: January, 2018
+# Author: gerald.venzl@oracle.com
```

Comment:
```suggestion
# Copyright (c) 2022 Oracle and/or its affiliates.
# Licensed under the Universal Permissive License v 1.0 as shown at
# https://oss.oracle.com/licenses/upl.
#
```

Reply:
Thank you for reviewing this so quickly! I'll make the requested changes as soon as I can. It may be this evening (in the US) before I can work on it.